assignments/prototype-refactor.js

```diff
@@ -7,3 +7,131 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+  
+/*
+  === GameObject ===
+  * createdAt
+  * name
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject{
+    constructor(obj) {
+      this.createdAt = obj.createdAt;
+      this.name = obj.name;
+      this.dimensions = obj.dimensions;
+        }
+        destroy(){
+            return `${this.name} was removed from the game.`;
+        }
+    
+    };
+    /*
+          === CharacterStats ===
+          * healthPoints
+          * takeDamage() // prototype method -> returns the string '<object name> took damage.'
+          * should inherit destroy() from GameObject's prototype
+        */
+    
+        class CharacterStats extends GameObject{
+        constructor(obj) {
```

Comment:
The spacing and indentation is uneven. Doesn't affect code execution, but does affect readability, which is very important. Don't forget to use that prettier!